specification/README.md

```diff
@@ -0,0 +1,13 @@
+# Ably Asset Tracking Specification
+
+## Overview
+
+This document is meant to be a formal, canonical specification describing the Ably Asset Tracking solution.
+
+## Error codes
+
+The list of Ably Asset Tracking specific error codes.
+
+| Code   | Name            | Description                                                                                                                                    |
+| ------ | --------------- | ---------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
...and...

```suggestion
| ---- | ---- | ----------- |
```

to match. 😁

Reply:
https://github.com/ably/ably-asset-tracking-common/pull/56/commits/bf751cb491e50ddbe68e18bd43b8da9a4b9a1c31